assets/scripts/info_bubble/__tests__/Variants.test.js

```diff
@@ -54,20 +54,21 @@ describe('Variants', () => {
         <Variants type={INFO_BUBBLE_TYPE_SEGMENT} position={0} />,
         { initialState }
       )
+
       expect(asFragment()).toMatchSnapshot()
     })
 
     it('handles switching segment variant', () => {
-      renderWithReduxAndIntl(
+      const container = renderWithReduxAndIntl(
```

Comment:
So I ran into problems here trying to not declare an object because there was no store. Also odd was that this didn't work if I declared it like `const { container } `  vs how I wrote it. 

So again, is the below sort of approach acceptable or am I missing the point?

Reply:
You'd want to destructure like so:

```js
const { store, getByTitle } = renderWithReduxAndIntl(
```

Otherwise what you're doing is just renaming `wrapper` to `container`.

Reply:
ahh that makes sense!